tests/apollo/test_skvbc_view_change.py

```diff
@@ -359,11 +360,136 @@ async def _wait_for_read_your_writes_success(self, tracker):
                     else:
                         break
 
+    @with_trio
+    @with_bft_network(start_replica_cmd,
+                      selected_configs=lambda n, f, c: f >= 2)
+    @with_constant_load
+    async def test_f_staggered_replicas_requesting_vc(self, bft_network, skvbc, nursery):
```

Comment:
As discussed offline, let's move this kind of tests into a dedicated Apollo module, called for example `test_skvbc_chaotic_startup.py`. Can you please also move there the test I added some time ago, which starts replicas with delays in-between:
https://github.com/vmware/concord-bft/blob/master/tests/apollo/test_skvbc.py#L219

Thanks!